index.html

```diff
@@ -496,21 +493,21 @@ <h2>
         unexpected behavior, use a scope ending in a <code>/</code>.
       </div>
       <p>
-        If the <a>application context</a>'s <a>active document</a>'s
-        [=Document/URL=] is not <a data-lt="within-scope-manifest">within
-        scope</a> of the <a>application context</a>'s manifest, the user agent
-        SHOULD show a prominent UI element indicating the [=Document/URL=] or
-        at least its <a>origin</a>, including whether it is served over a
-        secure connection. This UI SHOULD differ from any UI used when the
-        [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        If the [=application context=]'s [=active document=]'s [=Document/URL=]
+        is not [=processed manifest/within scope=] of the [=application
+        context=]'s [=processed manifest=], the user agent SHOULD show a
```

Comment:
I noticed that you changed all references to this from "of the application context's manifest" to "of the application context's processed manifest". That has two issues:

1. It's quite verbose to always have to say "processed manifest".
2. It implies that an application context has both an unprocessed and processed manifest applied to it, or that either an unprocessed or a processed manifest can be applied. I don't think it should be necessary to make this distinction.

I think it should be possible to say "within scope of the application context's manifest" and know that that unambiguously means the manifest has been processed. If you don't think it means that, perhaps we can change the definition of "application context" or "applied" to explicitly say that it's a processed manifest that's applied.

Reply:
I'll have another read... I see value in  having the distinction between the (unprocessed) manifest and the processed manifest, which is applied.

Reply:
I'll see if the unprocessed manifest idea can go away entirely... it's just JSON until it becomes a processed thing, so maybe there is a way to make it go away.